<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated" stdset="0">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/bg_main.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="go_new">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>145</width>
      <height>145</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    color: white;
    font: bold 10pt;
    border:none;
    min-height: 24px;
    min-width: 60px;
    background-color:rgba(255,255,255,0);
   	background-image :url(:/image/add.png) 
	
}
QPushButton:hover
{
    color: white;
    font: bold 10pt;
    border:none;
    min-height: 24px;
    min-width: 60px;
    background-color:rgba(255,255,255,0);
   	background-image :url(:/image/add_2.png) 
} 

QPushButton:pressed
{
    color: white;
    font: bold 10pt;
    border:none;
    min-height: 24px;
    min-width: 60px;
    background-color:rgba(255,255,255,0);
   background-image :url(:/image/add_2.png) 
} </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="go_look">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>145</width>
      <height>145</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    color: white;
    font: bold 10pt;
    border:none;
    min-height: 24px;
    min-width: 60px;
    background-color:rgba(255,255,255,0);
   	background-image :url(:/image/look.png) 
	
}
QPushButton:hover
{
    color: white;
    font: bold 10pt;
    border:none;
    min-height: 24px;
    min-width: 60px;
    background-color:rgba(255,255,255,0);
   	background-image :url(:/image/look_2.png) 
} 

QPushButton:pressed
{
    color: white;
    font: bold 10pt;
    border:none;
    min-height: 24px;
    min-width: 60px;
    background-color:rgba(255,255,255,0);
   background-image :url(:/image/look_2.png) 
} </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="join_activity">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>145</width>
      <height>145</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    color: white;
    font: bold 10pt;
    border:none;
    min-height: 24px;
    min-width: 60px;
    background-color:rgba(255,255,255,0);
   	background-image :url(:/image/join.png) 
	
}
QPushButton:hover
{
    color: white;
    font: bold 10pt;
    border:none;
    min-height: 24px;
    min-width: 60px;
    background-color:rgba(255,255,255,0);
   	background-image :url(:/image/join_2.png) 
} 

QPushButton:pressed
{
    color: white;
    font: bold 10pt;
    border:none;
    min-height: 24px;
    min-width: 60px;
    background-color:rgba(255,255,255,0);
   background-image :url(:/image/join_2.png) 
} </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="go_person">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>45</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    color: white;
    font: bold 10pt;
    border:none;

  
    background:url(:/image/profile.png) 
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="go_set">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>50</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    color: white;
    font: bold 10pt;
    border:none;

  
    background:url(:/image/setting.png) 
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>546</width>
      <height>67</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    color: white;
    font: bold 10pt;
    border:none;
    min-height: 24px;
    min-width: 60px;
    background:url(:/image/title.png) 
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_title" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>28</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_menu" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lab_Title">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 14pt &quot;Aharoni&quot;;
font: 75 9pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnMenu_Min">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>最小化</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnMenu_Close">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>关闭</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
